docs/environments/third_party_environments.md

```diff
@@ -42,240 +42,247 @@ A short 2 sentence description.
 
 - [gym-electric-motor: Gym environments for electric motor simulations](https://github.com/upb-lea/gym-electric-motor)
 
-An environment for simulating a wide variety of electric drives taking into account different types of electric motors and converters.
+  An environment for simulating a wide variety of electric drives taking into account different types of electric motors and converters.
 
 - [racecar_gym: Miniature racecar env using PyBullet](https://github.com/axelbr/racecar_gym/)
 
-A gym environment for a miniature racecar using the [PyBullet](https://github.com/bulletphysics/bullet3) physics engine.
+  A gym environment for a miniature racecar using the [PyBullet](https://github.com/bulletphysics/bullet3) physics engine.
 
 - [sumo-rl: Reinforcement Learning using SUMO traffic simulator](https://github.com/LucasAlegre/sumo-rl)
 
-![Gymnasium version dependency](https://img.shields.io/badge/Gymnasium-v0.26.3-blue)
-![GitHub stars](https://img.shields.io/github/stars/LucasAlegre/sumo-rl)
+  ![Gymnasium version dependency](https://img.shields.io/badge/Gymnasium-v0.26.3-blue)
+  ![GitHub stars](https://img.shields.io/github/stars/LucasAlegre/sumo-rl)
 
-Gymnasium wrapper for various environments in the SUMO traffic simulator. Supports both single and multiagent settings (using [pettingzoo](https://pettingzoo.farama.org/)).
+  Gymnasium wrapper for various environments in the SUMO traffic simulator. Supports both single and multiagent settings (using [pettingzoo](https://pettingzoo.farama.org/)).
 
 ### Biological / Medical environments
 *Interacting with Biological Systems.*
 
 - [ICU-Sepsis: A Benchmark MDP Built from Real Medical Data](https://github.com/icu-sepsis/icu-sepsis)
 
-![Gymnasium version dependency](https://img.shields.io/badge/Gymnasium-v0.28.1-blue)
-![GitHub stars](https://img.shields.io/github/stars/icu-sepsis/icu-sepsis)
+  ![Gymnasium version dependency](https://img.shields.io/badge/Gymnasium-v0.28.1-blue)
```

Comment:
you are adding a bunch of spaces at the start of lines for some reason

Reply:
<img width="638" alt="Screenshot 2024-09-26 at 3 02 37 PM" src="https://github.com/user-attachments/assets/6527e84f-1943-4cbf-8c1d-7d9a5fb9e772">

This makes the content properly indented under the bullet as opposed to outdenting to body level after each bullet item, but happy to revert if that is preferred.